std/fs/constants.ts

```diff
@@ -0,0 +1,6 @@
+const { build } = Deno;
+
+import { Eol } from "./eol.ts";
```

Comment:
You could do something like 

```typescript
import { EOL as EOL_ } from "./eol.ts";
// ...
export const EOL = isWindows ? EOL_.CRLF : EOL_.LF;
```

Reply:
But why is eol.ts a separate file even? It's not useful to have so many small files and redirection.

Reply:
They're both expected to be imported through `fs/mod.ts` so they need different names.

`isWindows` won't have another home but I guess the constant `EOL` could also go in `eol.ts`. This is just transferred from `path/constants.ts`.

Reply:
Ehh I think there should be only one "EOL" variable. Having "EOL" and "Eol" and "EOL_" is only confusing. Something needs to change here.

IMO `enum EOL { CRLF = "\r\n", LF = "\n" }` is a useless abstraction. We should just use the literal strings "\r\n" and "\n"